cdk/src/open-data-platform/data-plane/data-import/write-demographic-data.handler.ts

```diff
@@ -219,7 +293,43 @@ class DemographicsTableRowBuilder {
     return this;
   }
 
-  build(): DemographicsTableRow {
-    return this._row;
+  geom(geom: string): DemographicsTableRowBuilder {
+    this._row.geom = geom;
+    return this;
+  }
+
+  // TODO clarify these are the right types (double?)
+  build(): SqlParametersList {
+    return [
+      {
+        name: 'census_geo_id',
+        value: { stringValue: this._row.census_geo_id.toString() },
```

Comment:
How about the _row.census_geo_id is a string and when you build the row you do this conversion there?

Reply:
Will move this to when you build the row, but the field type actually is string so I guess parsing from the GeoJSON doesn't stringify all fields after all.